docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -9,94 +9,217 @@ keywords: [human tasks, orchestration, getting started, user guide]
 <span class="badge badge--beginner">Beginner</span>
 <span class="badge badge--medium">Time estimate: 15 minutes</span>
 
-Using [Camunda Platform 8](https://camunda.io), you can orchestrate human tasks by assigning them to users. Then, users can enter the necessary data to drive the business process.
+import ExpressionInputImg from './img/expression-input-example.png';
+import FormValuesImg from './img/form-values-example.png';
+import ImplementModeImg from './img/implement-mode-active.png';
+import FormLinkingImg from './img/form-linking.png';
+import ModelerNavImg from './img/modeler-navigation.png';
+import ModelerGlobalNavImg from './img/modeler-global-nav.png';
+import ModelerFormMenuImg from './img/modeler-form-menu.png';
+import RunProcessImg from './img/run-process.png';
+import OperateHumanTasks from './img/operate-human-tasks.png';
+import FormEditorImg from './img/form-editor.png';
+import NavigationHistoryImg from './img/modeler-navigation-history.png';
 
-When a process instance arrives at such a user task, a new job is created. The process instance stops at this point and waits until the job is completed. Applications like [Tasklist](/components/tasklist/introduction-to-tasklist.md) can be used by humans to complete these tasks.
+import clsx from "clsx";
 
-While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial.
+Camunda Platform 8 allows you to orchestrate processes with human tasks of any complexity. Utilizing user tasks, you can create and assign tasks to users. Then, users can perform their work and enter the necessary data to drive the business process.
 
-In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser. If you prefer a video-based learning experience, visit [this Camunda Academy course](https://bit.ly/3PJJocB) on Camunda Platform 8 and getting started with human workflows.
+This guide introduces you to the basics of human task orchestration. You will create a simple process to decide on an expense report, and drive the process flow according to that decision. This process is entirely executable in the browser.
```

Comment:
I was expecting to see an expense report process, but the contents of this guide show the "Whats for dinner?" example.

Reply:
Fixed in both versions.